CHANGELOG.md

```diff
@@ -38,6 +38,7 @@ This section is for maintaining a changelog for all breaking changes for the cli
 ### Removed
 
 ### Fixed
+- Fix for 'scroll' parameter not serialized to SearchRequest.
```

Comment:
```suggestion
- Fix for 'scroll' parameter not serialized to SearchRequest ([#677](https://github.com/opensearch-project/opensearch-java/pull/677))
```

Reply:
oh my god...   I had exactly that in my editor, but I somehow did not click "save" before committing (which IDE isn't auto-save nowadays?)